src/components/custom-field-input-text/custom-field-input-text.css

```diff
@@ -0,0 +1,84 @@
+@import '../../styles/colors.css';
+@import '../../styles/spacing.css';
+@import '../../styles/typography.css';
+
+.additional-information-icon {
```

Comment:
Question:

- Could we postpone shipping this additional info icon until we get a tooltip functionality?

Reply:
From our 1<>1: 

- Let's remove this icon as a commit and create a pivotal story to follow up with tooltip functionality (not for this version since we might be waiting on Mavengers for their tooltip work)

Reply:
https://www.pivotaltracker.com/story/show/170224608